### PR TITLE
add enterprise section

### PR DESCRIPTION
**Pull request by pablodanswer**
Moves enterprise features to new enterprise section

### CODE REVIEW

**Comment by pablodanswer** [comment]
Screenshot 2024-06-24 at 1 43 17 PM

Screenshot 2024-06-24 at 1 43 25 PM

File: enterprise/configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (yuhongsun96) [inline enterprise/configuration.mdx:1]: Let's use Enterprise Edition (or EE shorthand) across the board. We want to be consistent for this one where possible

File: enterprise/enterprise.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Enteprise Overview"
```
Comment (yuhongsun96) [inline enterprise/enterprise.mdx:2]: Same here

File: enterprise/enterprise.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+icon: "user-large"
+---
+
+
+At Danswer, we strive to make our core open-source offering as powerful as possible. However, we understand that large enterprises require additional support and functionality, so we offer additional enterprise-oriented features and functionality.
+
+## Enterprise Features
```
Comment (yuhongsun96) [inline enterprise/enterprise.mdx:9]: same

File: enterprise/configuration.mdx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+title: "Enteprise Configuration"
+icon: "gears"
+description: 'Enterprise-only configuration'
+---
+
+
+## Enterprise environment variables
```
Comment (yuhongsun96) [inline enterprise/configuration.mdx:8]: same here

File: mint.json [hunk 5]
```diff
@@ -74,6 +74,7 @@
         "production/digitalocean"
       ]
     },
+
```
Comment (yuhongsun96) [inline mint.json:77]: nit

**Pull request merged**